.github/workflows/publish-docker.yml

```diff
@@ -16,7 +16,7 @@ on:
 env:
   REGISTRY: ghcr.io
   TAG_NAME: ${{ github.event.inputs.TAG_NAME || github.event.release.tag_name }}
-  pathPrefix: ${{ github.event.inputs.pathPrefix }}
+  pathPrefix: ${{ github.event.inputs.pathPrefix || '/stac-browser/' }}
```

Comment:
Looks good. Did you validate that the result parses and resolves on nginx correctly?

I ask only out of paranoia, since I got a similar problem with double-quotes (https://github.com/crim-ca/stac-browser/commit/bb86c4e93bf4dd5d02a0bb9340f594d078cc0dcd)

Reply:
@fmigneault  do you mind if I manually re-run the docker build workflow while we're waiting to resolve this PR so that the latest docker image isn't broken?

Reply:
No problem. Go ahead.

Reply:
I reran it with explicit `/stac-browser/`. The default is in this branch, not the main branch.